Tests/Microsoft.AppCenter.Test.Windows/Storage/StorageAdapterTest.cs

```diff
@@ -5,63 +5,286 @@
+            }
+        }
 
-            // Should not crash even if directory is null.
-            adapter.InitializeStorageAsync().Wait();
+        [TestMethod]
+        public void DatabaseIsNotInitilazeWhenCallDelete()
+        {
+            // Prepare data.
+            var exception = new StorageException("The database wasn't initialized.");
+            try
+            {
+                // Try get data before database initialize.
+                _adapter.Delete(TableName, ColumnChannelName, new object[] { StorageTestChannelName });
+                Assert.Fail("Should have thrown exception");
+            }
+            catch (Exception e)
+            {
+                Assert.AreEqual(exception.Message, e.Message);
+            }
+            try
+            {
+                // Initialize database.
+                _adapter.Initialize(DatabasePath);
+            }
+            catch
+            {
+                // Handle exception, database is not created with Mock.
+            }
+            CreateTableHelper(_adapter);
+            try
+            {
+                // Try get data after database initialize.
+                _adapter.Delete(TableName, ColumnChannelName, new object[] { StorageTestChannelName });
+            }
+            catch (Exception e)
+            {
+                Assert.Fail("Shouldn't have thrown exception");
+            }
         }
 
         [TestMethod]
-        public void CreateStorageAdapterExceptionIsWrapped()
+        public void CreateTable()
         {
-            var adapter = new StorageAdapter("path/to/database/file.db")
+            // Prepare data.
+            try
+            {
+                _adapter.Initialize(DatabasePath);
+            }
+            catch
             {
-                _databaseDirectory = Mock.Of<Directory>()
-            };
-            var sourceException = new System.IO.PathTooLongException();
+                // Handle exception, database is not created with Mock.
+            }
+
+            // Create test table.
+            CreateTableHelper(_adapter);
+
+            // Insert test data.
+            InsertToTableHelper(_adapter);
+            var count = _adapter.Count(TableName, ColumnChannelName, StorageTestChannelName);
+            Assert.AreEqual(1, count);
+
+            // Verify.
+            var testEntries = _adapter.Select(TableName, ColumnChannelName, StorageTestChannelName, null, null).ToList();
+            Assert.AreEqual(1, testEntries.Count);
+            var entryId = 0L;
+            testEntries.ForEach(entry =>
+            {
+                entryId = (long)entry[0];
+                Assert.AreEqual(entry[1], StorageTestChannelName);
+                Assert.AreEqual(entry[2], "");
+            });
+            _adapter.Delete(TableName, ColumnIdName, new object[] { entryId });
+            count = _adapter.Count(TableName, ColumnChannelName, StorageTestChannelName);
+            Assert.AreEqual(count, 0);
```

Comment:
```suggestion
            Assert.AreEqual(0, count);
```
C# `Assert.AreEqual` first parameter is expected value, second is actual value.